redaxo/src/addons/metainfo/boot.php

```diff
@@ -44,3 +44,28 @@
 
     rex_extension::register('PAGE_CHECKED', 'rex_metainfo_extensions_handler');
 }
+
+rex_extension::register('EDITOR_URL', function (rex_extension_point $ep) {
+    if (!preg_match('@^rex:///metainfo/(\d+)@', $ep->getParam('file'), $match)) {
+        return;
+    }
+
+    $id = $match[1];
+    $sql = rex_sql::factory();
+    $sql->setQuery('SELECT `name` FROM '.rex::getTable('metainfo_field').' WHERE id = ? LIMIT 1', [$id]);
+
+    if (!$sql->getRows()) {
+        return;
```

Comment:
sollte man in den `return;` cases `$ep->getSubject()` zurückgeben?

Reply:
Bei den Extensions gewinnt immer die letzte. Wenn man `null` zurückgibt, wird der Wert nicht übernommen, sondern $subject wird erhalten (außer man ändert ihn halt über `setSubject`).
https://github.com/redaxo/redaxo/blob/c6f1fdb8a002850c3224149b6a53e5acccfd486e/redaxo/src/core/lib/extension.php#L46-L49